ERCS/erc-0000.md

```diff
@@ -0,0 +1,131 @@
+---
+eip: 0000
```

Comment:
```suggestion
eip: 8063
```
Assigning next sequential EIP/ERC/RIP number.
Numbers are assigned by editors & associates.

Please also update the filename.